client/src/modules/cash/modals/invoice-modal.html

```diff
@@ -14,7 +14,7 @@
 
   <!-- warn the client if not debtor id has been passed in -->
   <p ng-show="CashInvoiceModalCtrl.missingId" class="text-danger text-center">
-    <i class="fa fa-warning"></i> {{ ::"FORM.ERRORS.MISSING_DEBTOR_ID" | translate }}
+    <i class="fa fa-warning"></i> <span translate>{{ ::"FORM.ERRORS.MISSING_DEBTOR_ID" }}</span>
```

Comment:
This also does not need the `::`.  It can just be `<span translate>FORM.ERRORS.MISSING_DEBTOR_ID</span>`.